### PR TITLE
task completed

### PR DESCRIPTION
**Pull request by andrii-petruk**
- [DEMO LINK](https://andrii-petruk.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://andrii-petruk.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+.form__input_small::placeholder {
+  font-size: 14px;
+  line-height: 22px;
+}
+
+.form__input:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+  font-weight: 600;
```
Comment (ivanchuk-yurii) [inline src/style.css:108]: make `placeholder` bold on `:focus` also

**Review by ivanchuk-yurii — CHANGES_REQUESTED** [review]

**Comment by andrii-petruk** [comment]
please check please https://github.com/mate-academy/layout_search-bar-airbnb/pull/2486/commits/48dc86b8a1754782dd469cd996fa35f6145eea21

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+@font-face {
+  font-family: "Avenir";
+  src: url(/fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 600;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
```
Comment (stanislav-topikha) [inline src/style.css:20]: add fallback font

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+  padding: 0;
```
Comment (stanislav-topikha) [inline src/style.css:21]:
```suggestion
```

File: src/index.html [hunk 4]
```diff
@@ -11,6 +11,49 @@
...
+      data-qa="big"
+    >
+      <div class="form__div form__div_big">
+        <img 
+          class="form__icon"
+          src="images/Search.svg" 
+          alt="Search icon"
+        >
+        <input 
+          class="form__input form__input_big"
```
Comment (stanislav-topikha) [inline src/index.html:27]:
```suggestion
    <form 
      class="search__form"
      action="search"
      method="get"
      data-qa="big"
    >
      <div class="form__div">
        <img 
          class="form__icon"
          src="images/Search.svg" 
          alt="Search icon"
        >
        <input 
          class="form__input"
```
lets make it work like this, + remove repeated styles

**Review by stanislav-topikha — CHANGES_REQUESTED** [review]

**Review by liliia-livandovskaya — APPROVED** [review]
Great job! 🔥

**Comment by andrii-petruk** [comment]
> Great job! 🔥
Thanks !

**Pull request closed without merging**